### PR TITLE
Crud operations

### PR DESCRIPTION
**Pull request by ArjunAlluri**
Please review

### CODE REVIEW

File: backend/index.ts [hunk 1]
```diff
@@ -1,12 +1,82 @@
...
 const app = express();
+app.use(bodyParser.json());
+
+// Connect to MongoDB
+mongoose.connect(
+  'mongodb://localhost/mydatabase',
+  { useNewUrlParser: true }
+);
+
+const itemSchema = new mongoose.Schema({
```
Comment (GowthamChowta) [inline backend/index.ts:14]: Put the schema into a seperate file. 
Create a schema for Blog. 
Replace itemSchema with blogSchema.

File: backend/index.ts [hunk 2]
```diff
@@ -1,12 +1,82 @@
...
+  name: String,
+  description: String,
+});
 
-const port = process.env.PORT || 3001;
+const Item = mongoose.model('peoples', itemSchema);
 
-app.get('/', (req, res) => {
-  res.send('Dummy API');
+app.post('/add-item', async (req, res) => {
```
Comment (GowthamChowta) [inline backend/index.ts:21]: Follow the same notation 
Replace /add-item with -> /addBlog
/update-item with -> /updateBlog
Similarly /deleteBlog and /getBlog

File: backend/index.ts (hunk 1 above)
Comment (ArjunAlluri) [reply backend/index.ts:14]: What would the schema look like for a blog? 
I'm thinking title, date n time, author, text. anything else?

File: backend/index.ts (hunk 2 above)
Comment (ArjunAlluri) [reply backend/index.ts:21]: Got it.

File: backend/index.ts [hunk 3]
```diff
@@ -1,12 +1,82 @@
...
+  } catch (error) {
+    res.status(500).json({ message: 'Error adding item', error: error.message });
+  }
 });
 
-app.listen(port, () => {
-  console.log(`Server is running on port ${port}`);
-});
\ No newline at end of file
+app.put('/update-item/:id', async (req, res) => {
```
[the remark this answers is not in the document]
Comment (ArjunAlluri) [reply backend/index.ts:32]: I'll look into this

File: backend/index.ts [hunk 4]
```diff
@@ -1,12 +1,82 @@
...
+app.get('/get', async (req,res)=> {
+      try{
+        const allitems = await Item.find();
+        res.json(allitems);
+      }catch(error){
+        res.status(500).json({message: 'error fetching items', error: error.message});
+      }
+})
+
+app.delete('/remove/:id', async (req,res)=> {
```
Comment (GowthamChowta) [inline backend/index.ts:62]: This is nice. But ideally we donot want to delete anything from database. 
Can we add a new attribute that says isActive and put that flag to false. 

This way, if the user wants to restore his deleted blog, he can always come back. 

In all the API's we write at Amazon or old companies we don't delete anything. We just toggle some flag.

File: backend/index.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (ArjunAlluri) [reply backend/index.ts:32]: I am using express-validator library.

**Pull request closed without merging**